decidim-proposals/app/views/decidim/proposals/admin/proposals_valuator_mailer/notify_proposals_valuator.html.erb

```diff
@@ -0,0 +1,12 @@
+<p class="email-greeting"><%= t(".greeting", name: @valuator_user.name) %></p>
+
+<p class="email-instructions"><%= t(".email_body", admin_name: @admin.name) %></p>
+
+<p class="email-instructions">
+  <% @proposals.each do |proposal| %>
+    <%= sanitize link_to(t(".proposal_url", num: (@proposals.index(proposal) + 1).to_s),
+                         Decidim::ResourceLocatorPresenter.new(proposal).url), tags: ["a"], attributes: ["href"] %><br>
```

Comment:
```suggestion
    <%= sanitize link_to(translated_attribute(proposal.title), proposal_url(proposal)), tags: ["a"], attributes: ["href"] %><br>
```

- Let's use the proposal title instead of a numbering.

- We can use a helper instead of the class here, define the helper in `ProposalsValuatorMailer`

```ruby
module Decidim
  module Proposals
    module Admin
      class ProposalsValuatorMailer < Decidim::ApplicationMailer
        include Decidim::TranslationsHelper
        include Decidim::SanitizeHelper
        include Decidim::ApplicationHelper
        include Decidim::TranslatableAttributes

[...]

private

def proposal_url(proposal)
  Decidim::ResourceLocatorPresenter.new(proposal).url
end
```

Reply:
`proposal_url(proposal)`does not work in the view. I can't get it to work.